mteb/models/blip2_models.py

```diff
@@ -13,7 +13,7 @@
 
 
 def blip2_loader(**kwargs):
-    try:  # a temporal fix for the dependency issues of vista models.
+    try:  # a temporal fix for the dependency issues.
```

Comment:
genreally I would recommned adding these to the pyproject.toml as optional dependencies as this allow us to specify the version:

```pip install mteb[salesforce-lavis]```

Reply:
sure! good idea